src/roles/check_database_connection/tasks/check.yaml

```
@@ -0,0 +1,19 @@
+- name: Check {{ db_item.name }} database connectivity
```

Ansible lint doesn't like it if you use `{{ item }}` inside the `name` (at the end it's fine with it):
```
Jinja templates should only be at the end of 'name'
```

I must admit I am not a huge fan of this rule, so would be OK if we just disable it in our config.

OTOH we can rewrite this as:
```suggestion
- name: Check database connectivity to {{ db_item.name }}
```